### PR TITLE
Anshu-Assignment-19-40min+10+5+5+5-Done

### PR DESCRIPTION
**Pull request by AnshuKJun20**
27thProgram1
27thProgram2
27thProgram3
27thProgram4
27thPrograme5

### CODE REVIEW

File: src/anshuKJun20/PrimeNumber.java [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+	  
+	  for(int i=2; i<= itsSquareno; i++) {
+		  if(number%i == 0)
+		  {
+			  System.out.println(number+" is not a prime number");
+		  }
+	  }
+	  System.out.println(number+" is prime number");
+  }
+  void addPrimeNumber() {
```
Comment (KrishnaKTechnocredits) [inline src/anshuKJun20/PrimeNumber.java:15]: Why this separate method is required ? You should directly write count and some logic in findPrimeNo() method. Take two instance variable.

**Review by KrishnaKTechnocredits — CHANGES_REQUESTED** [review]
.

File: src/anshuKJun20/PrimeNumber.java (hunk 1 above)
Comment (KrishnaKTechnocredits) [inline src/anshuKJun20/PrimeNumber.java]: This method should be parameterized.  And use method findPrimeNo().

Comment (AnshuKJun20) [reply src/anshuKJun20/PrimeNumber.java]: Changes has been done. please Check it once

File: src/anshuKJun20/PrimeNumber.java [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+		for (int i = 2; i <= itsSquareno; i++) {
+			if (number % i == 0) {
+				System.out.println(number + " is not a prime number");
+			}
+		}
+		System.out.println(number + " is prime number");
+	}
+
+	void addPrimeNumber() {
```
Comment (KrishnaKTechnocredits) [inline src/anshuKJun20/PrimeNumber.java:15]: Still you have not made this method parameterized.

**Pull request merged**